rn/Teacher/src/common/formatters.js

```diff
@@ -127,9 +127,8 @@ export function formatStudentGrade (assignment: Assignment) {
 }
 
 export function personDisplayName (name, pronouns) {
-  let displayName = name
   if (pronouns != null) {
-    displayName += ` (${pronouns})`
+    return i18n('{name} ({pronouns})', { name, pronouns })
```

Comment:
the nice thing about the icu format is the named parameters already provide a helpful description of what this is doing, and make it unique from any other parenthetical.